packages/expo-av/android/src/test/java/expo/modules/av/progress/ProgressLooperTest.kt

```diff
@@ -0,0 +1,223 @@
+//    timeMachine.triggerListeners(1010)
+//    verify(exactly = 1) { callback() }
+//
+//    timeMachine.advanceBy(100)
+//    timeMachine.triggerListeners(1110)
+//    verify(exactly = 2) { callback() }
+//  }
+//
+//  @Test
+//  fun `callback invoked earlier even if interval lengthened`() {
+//    looper.loop(1000, callback)
+//
+//    looper.loop(2000, callback)
+//    timeMachine.advanceBy(1100)
+//    timeMachine.triggerListeners(1100)
+//
+//    verify(exactly = 1) { callback() }
+//  }
+//
+//  @Test
+//  fun `callback not invoked later even if interval lengthened`() {
+//    looper.loop(1000, callback)
+//
+//    looper.loop(2000, callback)
+//    timeMachine.advanceBy(1110)
+//    timeMachine.triggerListeners(1100)
+//
+//    verify(exactly = 1) { callback() }
+//
+//    timeMachine.advanceBy(1110)
+//    timeMachine.triggerListeners(2200)
+//
+//    verify(exactly = 1) { callback() }
+//  }
+//
+//  @Test
+//  fun `next tick scheduled with adjustment to passed time when invoked too late`() {
+//    looper.loop(1000L, callback)
+//
+//    timeMachine.advanceBy(1100)
+//    timeMachine.triggerListeners(1100)
+//
+//    verify(exactly = 1) { timeMachine.scheduleAt(900, any()) }
+//  }
+//
+//  @Test
+//  fun `next tick scheduled with adjustment to passed time when invoked too early`() {
+//    looper.loop(1000L, callback)
+//
+//    timeMachine.advanceBy(900)
+//    timeMachine.triggerListeners()
+//
+//    verify(exactly = 1) { timeMachine.scheduleAt(1100, any()) }
+//  }
+//
+//  @Test
+//  fun `old listener not notified after new is registered`() {
+//    looper.loop(1000, callback)
+//
+//    timeMachine.advanceBy(1100)
+//    timeMachine.triggerListeners()
+//    verify(exactly = 1) { callback() }
+//
+//    looper.changeListener { }
+//    timeMachine.advanceBy(1100)
+//    timeMachine.triggerListeners()
+//
+//    verify(exactly = 1) { callback() }
+//  }
+//
+//  @Test
+//  fun `new listener is notified after registration`() {
+//    looper.loop(1000) { }
+//
+//    timeMachine.advanceBy(1100)
+//    timeMachine.triggerListeners()
+//    looper.changeListener(callback)
+//    timeMachine.advanceBy(1100)
+//    timeMachine.triggerListeners()
+//
+//    verify(exactly = 1) { callback() }
+//  }
+//
+//  @Test
+//  fun `time machine not called if no looping started`() {
+//    timeMachine.advanceBy(1100)
+//    timeMachine.triggerListeners()
+//
+//    verify(exactly = 0) { timeMachine.scheduleAt(any(), any()) }
+//  }
+//
+//  @Test
+//  fun `time machine not called after looping stopped`() {
+//    looper.loop(1000) {}
+//    verify(exactly = 1) { timeMachine.scheduleAt(any(), any()) }
+//
+//    timeMachine.advanceBy(1100)
+//    timeMachine.triggerListeners()
+//    verify(exactly = 2) { timeMachine.scheduleAt(any(), any()) }
+//
+//    looper.stopLooping()
+//    timeMachine.advanceBy(100000)
+//    timeMachine.triggerListeners()
+//    verify(exactly = 2) { timeMachine.scheduleAt(any(), any()) }
+//  }
+//
+//}
```

Comment:
I don't get the idea of commenting out whole test file? 🤔 
Cannot it be done some other (more programmer-friendly) way?